app/assets/stylesheets/spree/frontend/base/_responsive.scss

```diff
@@ -0,0 +1,21 @@
+$tablet-screen: 660px;
```

Comment:
Is this a standard tabled width? I remember `768` being a common one, but I could be wrong.

Reply:
768px is a standard but some tablets starts even at 600px.

Reply:
Maybe I could lower it to 600px instead.

Reply:
If the layout works at that size go for it 👍